compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MainMethodGen.java

```diff
@@ -455,6 +458,8 @@ private void handleErrorFromFutureValue(MethodVisitor mv, boolean isTestFunction
         if (isTestFunction) {
             mv.visitFieldInsn(GETFIELD , FUTURE_VALUE , PANIC_FIELD, GET_THROWABLE);
```

Comment:
mv.visitFieldInsn(GETFIELD , FUTURE_VALUE , PANIC_FIELD, GET_THROWABLE) this is common to both if and else